specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-02-28-preview/Swagger/timeseriesinsights.json

```diff
@@ -1253,7 +1342,8 @@
         }
       },
       "required": [
-        "kind"
+        "kind",
+        "properties"
```

Comment:
Invalid definition: Where is this property called `properties`?

Reply:
You're right. It's the derived classes (EventHubEventSourceResource, IoTHubEventSourceResource) that have the properties collections.